source/MessageHandlers/PreviewHandler.ts

```diff
@@ -41,10 +42,10 @@ export default class PreviewHandler implements IMessageHandler {
         this.logQueries(integrationConfig.queries);
     }
 
-    public logQueries(queryStatements: string[]): void {
+    public logQueries(queryStatements: IQueryDefinition[]): void {
         this._logger.info('The following queries are configured to be run by the agent:');
         for (const query of queryStatements) {
```

Comment:
It's nitpicky, but it might be nice to rename this queryDefinition, since its semantics have changed a bit here